shell/assets/styles/themes/_light.scss

```diff
@@ -40,6 +40,8 @@ $contrasted-light: $lightest !default;
 // The terminal alway uses a light background, so okay to use a fixed color
 $selected: rgba(#3D98D3, .5);
 
+$drag-over: #DCDEE700;
```

Comment:
this is not a valid hex color code... Shouldn't it have only 6 chars?

Reply:
It was typo, sorry.